pkgs/development/compilers/zig/default.nix

```diff
@@ -0,0 +1,82 @@
+{
+  lib,
+  darwin,
+  callPackage,
+  llvmPackages_13,
+  llvmPackages_15,
+  llvmPackages_16,
+  llvmPackages_17,
+  llvmPackages_18,
+  zigVersions ? { },
+  isBootstrap ? false,
+}:
+let
+  versions = {
+    "0.9.0" = {
+      llvmPackages = llvmPackages_13;
+      hash = "sha256-x2c4c9RSrNWGqEngio4ArW7dJjW0gg+8nqBwPcR721k=";
+      bootstrapHashes = {
+        "linux-aarch64" = "sha256:00m6nxp64nf6pwq407by52l8i0f2m4mw6hj17jbjdjd267b6sgri";
```

Comment:
Can we use sri hashes here?

Reply:
SRI hashes with the bootstrap? I think I've tried that but the fetch builtin being used doesn't like that.

Reply:
Oh yeah, that's true. Do we need to use that or can we also use pkgs.fetchurl and the like?

Reply:
Looks like we might be able to use `pkgs.fetchurl` and when bootstrapping in stdenv, can use the boot one.